_includes/cockroachcloud/planning-your-serverless-cluster.md

```diff
@@ -0,0 +1,11 @@
+### Resource usage
+
+All resource usage in CockroachCloud Serverless (beta) is measured in Request Units, or RUs. RUs represent the compute and I/O resources used by a query. All database operations cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs. You can see how many request units your cluster has used on the [Cluster Overview](#view-cluster-overview) page.
+
+### Cluster scaling
+
+CockroachCloud Serverless (beta) clusters scale based on your workload. Free clusters include 250M RUs with 10M RUs reserved for [burst performance](architecture.html#concepts) and 5GiB of storage per month. They have a guaranteed baseline performance of 100 RUs per second, or up to 100 QPS.
+
+Paid clusters include additional resources with no throttling. You will only be charged for the resources you use up to your spend limit. If you reach your spend limit, your cluster will revert to the baseline performance of free clusters.
```

Comment:
I wouldn't say "no throttling" because you will still get throttled if you hit your spend limit.

Reply:
Another nit: "throttling" and "reverting to baseline performance" are actually the same thing. I think we should choose one phrase and use it everywhere. I prefer "reverting" because it sounds friendlier.

Reply:
we use "throttle" in the UI so it has to be mentioned in some places, but I can use "revert to baseline" when possible or change everything to "throttle"

Reply:
How does this sound: "Paid clusters include additional resources to maintain a high level of performance with larger workloads."

Reply:
> we use "throttle" in the UI so it has to be mentioned in some places, but I can use "revert to baseline" when possible or change everything to "throttle"

Understood. In that case I would personally just go with "throttle" everywhere.

> How does this sound: "Paid clusters include additional resources to maintain a high level of performance with larger workloads."

I think not bad, but the word "include" might be confusing because it kind of implies that performance is bundled into a fixed price. The way that I think of it is more like, "You can set your spend limit higher to maintain a high level of performance with larger workloads."